src/Events/Events.jsx

```diff
@@ -74,27 +49,37 @@ function convertEventResponseToModel(withDescription = false) {
 }
 
 @translate(['events'], { wait: true })
-export class Events extends Component {
+export default class Events extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
```

Comment:
You can also define the state property above the constructor. Something like;
```
class Events {
  state = {
    ...
  }
}
```
In that way you don't need the constructor.

Reply:
And about the `this.fetchEvents();`, maybe its better to put it into the `componentDidMount` lifecycle method. That would be the 'React way'.

Reply:
I am trying this for the Events page, but this must also be done for App.jsx and Contact.jsx.